src/client/components/SuggestedRoutes/SuggestedRouteCard.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import SuggestedRouteDetails from "./SuggestedRouteDetails";
+import SuggestedRouteSummary from "./SuggestedRouteSummary";
+
+export default function SuggestedRouteCard(props) {
```

Comment:
why not destructuring  the props here to make it shorter and more readable line?